xsd-cli/src/main/java/io/ballerina/xsd/cli/XsdCmd.java

```diff
@@ -66,13 +70,15 @@ public class XsdCmd implements BLauncherCmd {
     private static final String AUTO_GENERATED_MESSAGE = "// AUTO-GENERATED FILE. DO NOT MODIFY.\n" +
             "// This file is auto-generated by the Ballerina XSD tool.";
     public static final String TYPES_FILE_NAME = "types.bal";
+    public static final String OUTPUT_FILE_NAME = "types";
+    public static final String OUTPUT_FILE_EXTENSION = ".bal";
     private final PrintStream outStream;
     private final boolean exitWhenFinish;
     @CommandLine.Option(names = {"-h", "--help"}, hidden = true)
     private boolean helpFlag;
 
     @CommandLine.Parameters(description = "Input file path of the XSD schema")
-    private final List<String> argList = new ArrayList<>();
+    private List<String> argList = new ArrayList<>();
```

Comment:
Why are we allowing a list here? Can't we allow only one string value

Reply:
Fixed in [f8f97d4](https://github.com/ballerina-platform/xsd-tools/pull/13/commits/f8f97d43f3a9500cbe031a15a0160ec62dcb9002)